recipes/jinja2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.8" %}
+
+package:
+  name: jinja2
+  version: {{ version }}
+
+source:
+  fn: Jinja2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
+  md5: edb51693fe22c53cee5403775c71a99e
+
+build:
+  number:  0
+  script: python setup.py install --old-and-unmanageable
```

Comment:
Question are we going for @minrk suggestion to use `pip` or `--old-and-unmanageable`? or the current `python setup.py install --single-version-externally-managed --record record.txt`? It seems that mixing these will be a source of confusing in the future.